Modules/LearningSequence/classes/Player/class.ilLSPlayer.php

```diff
@@ -92,11 +100,15 @@ function ($item) {
                 return 'EXIT::' . $command;
             case self::LSO_CMD_NEXT:
                 $next_item = $this->getNextItem($current_item, $param);
+                if ($next_item->getAvailability() !== \ILIAS\UI\Component\Listing\Workflow\Step::AVAILABLE) {
+                    $next_item = $current_item;
+                }
                 break;
             case self::LSO_CMD_GOTO:
                 list($position, $next_item) = $this->findItemByRefId($param);
                 break;
-            default: //view-internal / unknown command
+            case self::LSO_CMD_CONTINUE:
```

Comment:
Where does this new command originate from? I do not see that we set this somewhere...

Reply:
sorry, this was a fragment I forgot to remove.